### ISSUE TITLE
Is It Possibile to dynamically add users to a room?

### ISSUE THREAD

**Issue by RizzuDev** [flyerhq/flutter_firebase_chat_core#91]
I would like to know if it is possible to dynamically add users (clearly knowing their IDs) to rooms. Thank you

**Comment by demchenkoalex** [comment]
I don't think it is in the lib, but you can just do it yourself I guess? Update room's users

**Comment by RizzuDev** [comment]
When I do it, it makes all the groups disappear.

**Comment by demchenkoalex** [comment]
there are some sorting logic in the lib you might want to check, I guess users are sorted alphabetically or something like that, don't remember, it was a PR

**Comment by RizzuDev** [comment]
Could you please help me? Because every time I try to update the rooms/ I can't find any room for my user, like I'm breaking something

**Comment by demchenkoalex** [comment]
Just take a look at the code, there is not much going on really :)
